search-app/src/cmr/search/services/acls/granule_acls.clj

```diff
@@ -152,22 +152,25 @@
         (gc/or-conds [concept-ids-cond entry-titles-cond])
         (or concept-ids-cond entry-titles-cond)))))
 
-
 (defn collection-identifier->query-condition
   "Converts an acl collection identifier to an query condition. Switches implementations based
   on whether the user's query contained collection ids. This implementation assumes the query-coll-ids
   passed in are limited to the provider of the collection identifier."
   [context query-coll-ids provider-id collection-identifier]
   (let [colls-in-prov-cond (provider->collection-condition query-coll-ids provider-id)]
-    (if-let [{:keys [entry-titles access-value temporal]} collection-identifier]
+    (if-let [{:keys [concept-ids entry-titles access-value temporal]} collection-identifier]
       (let [entry-titles-cond (provider+entry-titles->collection-condition
```

Comment:
Why are we keeping the entry-titles-cond? Shouldn't every ACL now have concept-ids in them? I thought that was something that was done with the ECHO-ACL/CMR ACL sync functionality.

Is it still possible that an ACL could be saved with the concept-ids?